### PR TITLE
groupnorm: eager static quant support

### PR DESCRIPTION
**Pull request by vkuzo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #39413 graph mode: util for fusion of functions which require observation
* #39203 quant layer/group/instance norm: make weights and biases optional
* #39096 instancenorm: static quant graph mode support
* #39095 groupnorm: graph mode static quant support
* #39094 layernorm: eager mode qat support
* #39093 instancenorm: eager mode QAT support
* #39092 groupnorm: eager mode QAT support
* #39091 instancenorm: eager static quant support
* **#39090 groupnorm: eager static quant support**
* #37802 quant: clean up normalization channels_last handling

Summary:

Makes quantized GroupNorm work in eager mode post training static quant.

Test Plan:

```
python test/test_quantization.py TestPostTrainingStatic.test_normalization
python test/test_quantization.py TestStaticQuantizedModule.test_group_norm
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21885262](https://our.internmc.facebook.com/intern/diff/D21885262)

### CODE REVIEW

**Review by supriyar — APPROVED** [review]
lgtm!

File: torch/nn/quantized/modules/normalization.py [hunk 1]
```diff
@@ -87,3 +87,69 @@ def from_float(cls, mod):
...
+        >>> # Separate 6 channels into 6 groups (equivalent with InstanceNorm)
+        >>> m = nn.GroupNorm(6, 6)
+        >>> # Put all 6 channels into a single group (equivalent with LayerNorm)
+        >>> m = nn.GroupNorm(1, 6)
+        >>> # Activating the module
+        >>> output = m(input)
+    """
+    __constants__ = ['num_groups', 'num_channels', 'eps', 'affine']
+
+    def __init__(self, num_groups, num_channels, weight, bias, scale, zero_point, eps=1e-5, affine=True):
```
Comment (supriyar) [inline torch/nn/quantized/modules/normalization.py:133]: Should `affine` default to true for the quantized module? It seems to toggle learnable parameter so doesn't look like we would use it.

Comment (vkuzo) [reply torch/nn/quantized/modules/normalization.py:133]: `affine` is used as a flag on whether the weight+bias tensors should exist: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/normalization.py.  The actual value of it is not used in the context of quantization, so keeping it just for consistency of init args between the FP vs Q modules.  The values of weight+bias tensors are what's used.  Flexible - lmk if something different would be better.
